paths_cli/compiling/tools.py

```diff
@@ -18,6 +18,10 @@ def custom_eval(obj, named_objs=None):
     }
     return eval(string, namespace)
 
+def custom_eval_int(obj, named_objs=None):
+    val = custom_eval(obj, named_objs)
+    return int(val)
+
```

Comment:
Most uses I saw assume this returns a positive int. Should this eval check for that, or should the code that use this check for that?

Reply:
No; in principle negative int is completely possible. Example: perhaps an input specifies a Python-style index, and use wants to select the last one (`-1`).

It might be reasonable to add "positive" or "nonnegative" variants, though. That can come later as needed.

Reply:
So here is a quick list of were nonpositive ints would break code:

paths_cli/compiling/engines.py:40:    Parameter('n_steps_per_frame', custom_eval_int,
paths_cli/compiling/engines.py:42:    Parameter("n_frames_max", custom_eval_int,

Will make a comment on those code lines as well

Reply:
Added `custom_eval_int_strict_pos` for this